demo/src/content-lists/content-tree-page.js

```diff
@@ -193,7 +193,7 @@ class ListsPage {
       this.#treeNavigationEl.replaceChildren(...parseHtml(`
       <button data-navigation-idx="0">Home</button>
       ${this.#traversalStack.slice(1).map((step, idx) => `
-      <span>&gt;</span>
+      <i class="material-icons"></i>
```

Comment:
Since there is an element here I would suggest this change :

```suggestion
      <span class="material-icons-outlined">chevron_right</span>
```

Reply:
Thank you for the suggestion, I updated the PR, however I still use the _i_ tag. If it's ok for you I'll create another PR to use the _i_ instead of a _span_. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i

Reply:
Good for me !